powerquery-docs/connectors/cloudbluepsa.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Power Query CloudBlue PSA connector
+description: Provides basic information and prerequisites for the operation of connector in conjuction with the API.
+author: bezhan-msft
```

Comment:
This should be your (CloudBlue) GitHub author.

Reply:
amended